.github/workflows/inductor-perf-test-nightly-aarch64.yml

```diff
@@ -27,6 +27,11 @@ on:
         required: false
         type: boolean
         default: true
+      default_cpu_f32:
```

Comment:
Just want to call out that GitHub has a limit on the number of inputs here.  The max is 10, which is not that many.  So, if you might want to have this as a more generic field, i.e. `cpu_configs`, and set the value to `default_cpu_f32` instead of using a boolean here.